about.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="Css/style.css">
+    <link rel="stylesheet" href="Css/about.css">
+    <title>GSN ABOUT </title>
+</head>
+<body>
+    <header>
+        <nav class="mobile">
+            <a href="#">
+                <div class="shamburger">
+                    <span class="bars"></span>
+                    <span class="bars"></span>
+                    <span class="bars"></span>
+                </div>
+            </a>
+            <ul class="disappears" id="navList2">
+                <li><a href="index.html">Home</a></li>
+                <li><a href="about.html">About Us</a> </li>
```

Comment:
- [ ] [OPTIONAL] It would be nice to remove the about page link when you are already in the about page.